src/tf/TF2Client.js

```diff
@@ -0,0 +1,182 @@
+/**
+ * @fileOverview
+ * @author David Gossow - dgossow@willowgarage.com
+ */
+
+import Action from '../core/Action.js';
+import Transform from '../math/Transform.js';
+
+import Ros from '../core/Ros.js';
+import {EventEmitter} from 'eventemitter3';
+
+/**
+ * A TF Client that listens to TFs from tf2_web_republisher.
+ */
+export default class TF2Client extends EventEmitter {
```

Comment:
I think this needs to be called something like `ROS2TFClient`, since "TF2" works on both ROS1 and ROS2.